src/components/Minibrowser/components/MinibrowserBreadcrumbs.vue

```diff
@@ -0,0 +1,102 @@
+<script>
```

Comment:
Is this component it will be in 'render function'?

Reply:
For now, I used the render function because I believe it's better to handle with the logic and I keep in a `.vue` file because of Vetur intellisense.